src/User32.Desktop/User32.cs

```diff
@@ -13,6 +13,8 @@ namespace PInvoke
     [OfferFriendlyOverloads]
     public static partial class User32
     {
+        public const int WHEEL_DELTA = 120;
```

Comment:
This is a User interface only constant. Look at this: [MSDN GET_WHEEL_DELTA_WPARAM](https://msdn.microsoft.com/en-us/library/windows/desktop/ms646254(v=vs.85).aspx). It returns the wheel movement in fractions of this constant. It is not defined anywhere on windows headers but it used all around mouse functions on User32 UI functions. So it is a facilitator for people to calculate stuff.